code/game/turfs/simulated/water.dm

```diff
@@ -0,0 +1,13 @@
+/turf/open/water
```

Comment:
We could make them the same path and add stuff/more depth to it. But whatever, that can easily be done in the future so you don't have to edit an infinite number of maps in this pr.